public/app/features/dashboard/components/PanelEditor/OptionsPaneItemDescriptor.tsx

```diff
@@ -35,10 +35,10 @@ export class OptionsPaneItemDescriptor {
   props: OptionsPaneItemInfo;
 
   constructor(props: OptionsPaneItemInfo) {
-      this.props = { ...props, id: props.id ?? props.title };      
-      if (this.props.id === '') {
-         this.props.id = uniqueId();                              
-      }              
+    this.props = { ...props, id: props.id ?? props.title };
```

Comment:
Suuuuuuper nit thing: This can be simplified even more using the plain ol' OR operator 😄:  `props.id || props.title || uniqueId()`.

Feel free to ignore me tho'. It's for the lolz.